src/app/home/home-routing.module.ts

```diff
@@ -1,13 +1,31 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { HomeComponent } from './home.component';
-import {AuthGuard} from '../auth/auth.guard';
+import { AuthGuard } from '../auth/auth.guard';
+import { Phase2Component} from './phase2/phase2.component';
+import { Phase3Component } from './phase3/phase3.component';
+import { Phase4Component } from './phase4/phase4.component';
 
 const routes: Routes = [
   {
     path: '',
     component: HomeComponent,
     canActivate: [AuthGuard],
+  },
+  {
+    path: 'phase2',
+    component: Phase2Component,
+    canActivate: [AuthGuard],
+  },
+  {
+    path: 'phase3',
```

Comment:
Routing to phase 3 isnt working on my side? Is it implemented yet?

Reply:
Phase3 not implemented for routing